examples/org-viewer-react/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "org-viewer-react",
+  "main": "dist/index.js",
+  "version": "0.2.0",
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "dev": "yarn clean && webpack-dev-server",
+    "build": "yarn clean && NODE_ENV=production webpack",
+    "build:stats": "yarn clean && webpack --env production --json > webpack-stats.json",
+    "clean": "rm -rf ./dist ./tsconfig.tsbuildinfo",
+    "compile": "tsc"
+  },
+  "dependencies": {
+    "@aragon/connect-react": "*",
+    "@emotion/core": "^10.0.28",
```

Comment:
How is `emotion` btw? Is it smaller than `styled-components`?

Reply:
> How is `emotion` btw?

It works fine, not that different from `styled-components` so far. I haven’t had a close look at the generated CSS code yet.

One nice thing is that we can use it with the `@jsx` pragma, to get a `css` prop without a babel plugin. It is a bit more verbose than the styled babel plugin (`css={css``}` + the import + the pragma directive), but I find it acceptable.

> Is it smaller than `styled-components`?

Yes, 6kb vs. 12kb:

- https://bundlephobia.com/result?p=@emotion/core@10.0.28
- https://bundlephobia.com/result?p=styled-components@5.1.1